components/display/ili9341.rst

```diff
@@ -0,0 +1,123 @@
+ILI9341 TFT LCD
+===============
+
+.. seo::
+    :description: Instructions for setting up ILI9341 TFT LCD display drivers.
+    :image: 24-ili9341-full.jpg
+
+.. _ili9341:
+
+Usage
+-----
+
+The ``ili9341`` display platform allows you to use
+ILI9341 (`datasheet <https://cdn-shop.adafruit.com/datasheets/ILI9341.pdf>`__,
+`Ali model used <https://s.click.aliexpress.com/e/_dTKH6Mt>`__)
+displays with ESPHome. Note that this display controller uses a lot of pins even if touch
+is not used. So it's better to used it with an EPS32 though esp8266 might be suitable
+too (with some SPI flash changes). 
+
+.. figure:: images/24-ili9341-full.jpg
+    :align: center
+    :width: 75.0%
+
+    ILI9341 common display
+
+
+.. code-block:: yaml
+
+    # Example minimal configuration entry
+    display:
+      - platform: ili9341
+        model: TFT_2.4
+        cs_pin: 14
+        dc_pin: 27
+        # led_pin: 32 // you may use it with a constant brightness but there is a better way
+        reset_pin: 33
+        rotation: 0
+        
+        lambda: |-
+          it.fill(ili9341::BLACK);
+          it.print(0, 0, id(font), "Hello World!");
+    
+Configuration variables
+***********************
+
+- **led_pin** (:ref:`Pin Schema <config-pin_schema>`): The display's backlight pin.
```

Comment:
mark this pin optional

Reply:
Led pin cannot be optional if this pin is not connected to a + rail the screen will be black all the time also it can be connected to a pwm then we can even dim the brightness of the screen

Reply:
I have this pin connected to +VCC via a R 3.3K and it works ok, this saves a pin on the ESP, may be this can be stated on the docs.